neurom/io/utils.py

```diff
@@ -115,6 +116,16 @@ def get_files_by_path(path):
 
 def load_neuron(handle, reader=None):
     """Build section trees from an h5 or swc file."""
+    if isinstance(handle, (morphio.Morphology, morphio.mut.Morphology)):
+        fd, temp_file = tempfile.mkstemp(str(uuid.uuid4()) + '.h5', prefix='neurom-')
+        os.close(fd)
+
+        # TODO: make a better morphio loader that does not require a tempfile
+        if isinstance(handle, morphio.Morphology):
+            handle = handle.as_mutable()
```

Comment:
Probably just my ignorance about morphio, but these two lines seem unnecessary to me

Reply:
morphio has 2 classes:
- `morphio.Morphology` 
- `morphio.mut.Morphology`

This line is to make sure we end up with a `morphio.mut.Morphology` regardless of the input class type of `handle` (`morphio.Morphology` or `morphio.mut.Morphology`)